app/controllers/jobseekers/saved_jobs_controller.rb

```diff
@@ -4,7 +4,6 @@ class Jobseekers::SavedJobsController < Jobseekers::BaseController
   # This action is not 'create' because we need to redirect here when an unauthenticated jobseeker attempts to save a job
   def new
     saved_job.save
-    request_event.trigger(:vacancy_save_to_account_clicked, vacancy_id: vacancy.id)
```

Comment:
Can you check if @stevenleggdfe is interested in whether the click event resulted in an actual save event?

Reply:
We already have all that information from page visit events and CRUD events